frontend/src/pages/news/components/newsList.tsx

```diff
@@ -0,0 +1,24 @@
+import { Heading, Text, Grid } from '@chakra-ui/react'
+import { NewsPreviewDTO } from '../../../util/views/news.view'
+import NewsListItem from './newsListItem'
+
+interface NewsListProps {
+  newsList: NewsPreviewDTO[]
+  warningMessage: String
+}
+
+const NewsList = ({ newsList, warningMessage }: NewsListProps) => {
+  return (
+    <>
+      <Heading>Hírek</Heading>
+      <Text>{warningMessage}</Text>
+      <Grid templateColumns={{ base: 'repeat(1, auto)', md: 'repeat(2, auto)' }} gap={10} marginTop={10}>
+        {newsList.map((n: NewsPreviewDTO, i) => (
+          <NewsListItem news={n} index={i} />
+        ))}
```

Comment:
The key should be here, on NewsListItem. Also, you shouldn't use index as a key.